src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -271,15 +272,27 @@ const WorkspaceDashboard = _.flow(
 
   const loadStorageCost = withErrorReporting('Error loading storage cost data', async () => {
     if (Utils.canWrite(accessLevel)) {
-      const { estimate, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
-      setStorageCost({ estimate, lastUpdated })
+      try {
+        const { estimate, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
+        setStorageCost({ estimate, lastUpdated })
+      } catch (error) {
+        error.status === 404 ?
+          setStorageCost({ isSuccess: false, estimate: 'Not available' }) :
+          __throw(error)
```

Comment:
```suggestion
        if (error.status === 404) {
          setStorageCost({ isSuccess: false, estimate: 'Not available' })
        } else {
          throw error
        }
```

Same comment for `loadBucketSize`.